packages/integration-tests/src/support/variables/cluster-details.ts

```diff
@@ -1,3 +1,9 @@
+import * as semver from 'semver';
```

Comment:
Is this package installed in the project dependencies?

Reply:
yes, it appears in yarn.lock

Reply:
no, is not installed. At the moment I can delete and then if we need we can add it